packages/skeleton-react/src/App.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { describe, expect, it } from "vitest";
+import { render } from "@testing-library/react";
+import App from "./App";
+
+describe("App", () => {
+  it("renders the component", () => {
+    const component = render(<App />);
+    expect(component).toBeTruthy();
```

Comment:
I know the example I shared with you didn't do this but using the `toBeInTheDocument()` asseartion appears to be a cleaner way to handle tests like this.
`toBeVisible()` is also a great assertion to use in such a usecase

Reply:
@TazorDE I'm down to implement these, unfortunately neither assertions appear to exist. Can you confirm how these should be setup and used?